jac-uk/apply
# 667 - separating QT code

**Author checklist**

- [x] Include primary ticket number in title

---

## What's included?
- Links to the Online Tests page should redirect to the new Online Tests website
- The existing Online tests page (on the Apply site) should still be accessible via its URL for now.
- The Online Tests page (and related components) have been deleted from the Apply site as part of this piece of development.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Confirm that the Online Tests menu item redirects to the new website, instead of a local webpage
- Check that the /online-tests page can still be accessed on the apply site directly if the URL is entered manually into the browser's address bar

When testing, developers should checkout the qt-separation branch on the apply repo at the same time as the setup-beta-site branch on online-tests repo and run both sites at the same time. They should find that their browser switches between the two sites seamlessly:
- Apply = http://localhost:8081
- Online Tests = http://localhost:8083

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
